pkgs/top-level/all-packages.nix

```
@@ -28404,6 +28404,8 @@ with pkgs;
 
   goimports-reviser = callPackage ../development/tools/goimports-reviser { };
 
+  gomarkdoc = callPackage ../development/tools/gomarkdoc {};
```

```suggestion
  gomarkdoc = callPackage ../development/tools/gomarkdoc { };
````
formatting nit. 

you don't have to do this now but in the future you can just put the package into `pkgs/by-name` and it will be picked up automatically